recipe/meta.yaml

```diff
@@ -201,6 +201,82 @@ outputs:
       home: https://gcc.gnu.org/onlinedocs/gccint/Libgcc.html
       license: GPL
 
+  - name: libgcc-ng
+    target: {{ ctng_target_platform }}
+    script: install-libgcc-no-gomp.sh
+    requirements:
+      run:
+        - _libgcc_mutex * main
+        - {{ pin_subpackage("openmp_impl", max_pin='x.x') }}
+    build:
+      number: {{ build_num }}
+      detect_binary_files_with_prefix: False
+      binary_relocation: False
+      missing_dso_whitelist:
+        - "*"
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libgcc_s.so
+        - test -f ${PREFIX}/lib/libgomp.so
+        - test -f ${PREFIX}/*/sysroot/lib/libgomp.so
+    about:
+      summary: The GCC low-level runtime library
+      home: https://gcc.gnu.org/onlinedocs/gccint/Libgcc.html
+      license: GPL
+
+  - name: libgomp
+    version: {{ libgomp_ver }}
+    target: {{ ctng_target_platform }}
+    script: install-libgomp.sh
+    requirements:
+      host:
+        - {{ pin_subpackage('libgcc-ng-build', exact=True) }}
```

Comment:
Since this package is only used here, why do we need separate package?

Reply:
Right yeah. The first way intried to split the build of lib gcc involved running the build steps for libgomp in a separate script. This worked in terms of getting the binary packages, but then the test that compiles FORTRAN code with openmp would not pass. I couldn’t figure out why. 

Thus I decided on the approach here to see if I could get things to work. Here I actually build libgcc twice.  The first of these is the regular build with libgomp bundled. The second is same build, but then I remove the parts of libgomp I don’t need at the end. Finally, I then build the libgomp package from the regular build by copying the file. This setup is a bit more convoluted but now the FORTRAN openmp code compiles properly. 

I still need to push a few changes to the symlinks since they are not quite right in libgcc right now. Hopefully that doesn’t break anything. 

I’m more than happy to hear suggestions on how to do this better as always!

I don’t know these builds very well and this is the only way I could get it to work so far.

Reply:
Right yeah. The first way intried to split the build of lib gcc involved running the build steps for libgomp in a separate script. This worked in terms of getting the binary packages, but then the test that compiles FORTRAN code with openmp would not pass. I couldn’t figure out why. 

Thus I decided on the approach here to see if I could get things to work. Here I actually build libgcc twice.  The first of these is the regular build with libgomp bundled. The second is same build, but then I remove the parts of libgomp I don’t need at the end. Finally, I then build the libgomp package from the regular build by copying the file. This setup is a bit more convoluted but now the FORTRAN openmp code compiles properly. 

I still need to push a few changes to the symlinks since they are not quite right in libgcc right now. Hopefully that doesn’t break anything. 

I’m more than happy to hear suggestions on how to do this better as always!

I don’t know these builds very well and this is the only way I could get it to work so far.

Reply:
OK I am trying a version with no extra package, but still taking the two build approach.